src/tls/cockpit-certificate-helper

```diff
@@ -0,0 +1,43 @@
+#!/bin/sh -eu
```

Comment:
nitpick: Please use `set -eu` on the second line, so that this stays around if you call the script through e.g. `sh -x helper` (which is fairly common for development)